drivers/exec/driver_test.go

```diff
@@ -870,10 +871,59 @@ func TestDriver_Config_validate(t *testing.T) {
 			}).validate())
 		}
 	})
+
+	t.Run("denied_host_ids", func(t *testing.T) {
+		invalidUidRange := "invalid denied_host_uids value"
+		invalidGidRange := "invalid denied_host_gids value"
+
+		for _, tc := range []struct {
+			uidRanges string
+			gidRanges string
+			errorStr  *string
+		}{
+			{uidRanges: "", gidRanges: "", errorStr: nil},
+			{uidRanges: "1-10", gidRanges: "1-10", errorStr: nil},
+			{uidRanges: "10-1", gidRanges: "", errorStr: &invalidUidRange},
+			{uidRanges: "", gidRanges: "10-1", errorStr: &invalidGidRange},
+		} {
+
+			validationErr := (&Config{
+				DefaultModePID: "private",
+				DefaultModeIPC: "private",
+				DeniedHostUids: tc.uidRanges,
+				DeniedHostGids: tc.gidRanges,
+			}).validate()
+
+			if tc.errorStr == nil {
+				require.Nil(t, validationErr)
+			} else {
+				require.Contains(t, validationErr.Error(), *tc.errorStr)
+			}
+		}
+	})
 }
 
 func TestDriver_TaskConfig_validate(t *testing.T) {
 	ci.Parallel(t)
+
+	current, err := users.Current()
+	require.NoError(t, err)
+	currentUid, err := strconv.ParseUint(current.Uid, 10, 32)
+	require.NoError(t, err)
+	nobody, err := users.Lookup("nobody")
+	require.NoError(t, err)
+	nobodyUid, err := strconv.ParseUint(nobody.Uid, 10, 32)
+	require.NoError(t, err)
```

Comment:
you can use `LookupUser` to make this less terrible - (exec is linux only it's fine)

https://github.com/hashicorp/nomad/blob/main/helper/users/lookup.go#L27-L31